.github/workflows/test.yaml

```diff
@@ -37,7 +37,7 @@ jobs:
         run : docker-compose down
   license-check:
     runs-on: ubuntu-latest
-    container: golang:1.15
+    container: golang:1.16
     steps:
     - uses: actions/checkout@v2
     - name: Download License Utility
```

Comment:
should we `go install` here since we're on 1.16 now?

Reply:
(referencing the line below the one I commented on)

Reply:
Nice catch!